systemtests/volplugin_test.go

```diff
@@ -102,6 +102,7 @@ func (s *systemtestSuite) TestVolpluginCrashRestart(c *C) {
 	time.Sleep(time.Second)
 	c.Assert(startVolplugin(s.vagrant.GetNode("mon0")), IsNil)
 	c.Assert(waitForVolplugin(s.vagrant.GetNode("mon0")), IsNil)
+	time.Sleep(30 * time.Second)
```

Comment:
this should not be necessary, but I know what's probably causing this; let's look at why it is later and see how we can reduce this if possible.